### PR TITLE
Cow 2

### PR DESCRIPTION
**Pull request by biocom**
Proper styling of primary/light buttons.

### CODE REVIEW

**Review by anxolin — COMMENTED** [review]
This is the perfect size of PR :) 

All good, but only the buttons for GP theme lost their round corners:
image
image

**Comment by biocom** [comment]
> This is the perfect size of PR :)
> 
> All good, but only the buttons for GP theme lost their round corners:
> image
> image

Should be addressed now.

File: src/custom/components/Button/index.tsx [hunk 1]
```diff
@@ -22,56 +22,82 @@ import {
...
+  }
 
   &:focus,
   &:hover,
   &:active {
-    ${({ theme }) => theme.bgLinearGradient}
-    background-color: transparent;
-    border: 0;
+    ${({ theme }) => (theme.buttonPrimary?.background ? theme.buttonPrimary?.background : theme.bgLinearGradient)}
+    border: ${({ theme }) => (theme.buttonPrimary?.border ? theme.buttonPrimary.border : 0)};
```
Comment (W3stside) [inline src/custom/components/Button/index.tsx:44]: no need for ternary logic its just overcrowded. set the default amount in the params (this comment applies to all of these)

Comment (biocom) [reply src/custom/components/Button/index.tsx:44]: But the params are set individually in each theme. Would you suggest to create a 'general' params. object/file to set the standards for all themes? Then if the theme provides a variable, it simply overrides the general parameter.

File: src/custom/pages/AppBody/index.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ export const BodyWrapper = styled(BodyWrapperMod)`
   border: ${({ theme }) => theme.appBody.border};
   padding: ${({ theme }) => theme.appBody.padding};
 `
-/**
+/**x
```
Comment (W3stside) [inline src/custom/pages/AppBody/index.tsx:12]: ?

Comment (biocom) [reply src/custom/pages/AppBody/index.tsx]: Updated. Probably fat finger on merge conflict.

File: src/custom/components/Button/index.tsx (hunk 1 above)
Comment (anxolin) [reply src/custom/components/Button/index.tsx:44]: Alternativelly, there could be a "baseTheme" we have, and then all themes inherit for it. Then this props would be always present (so we wouldn't need ternary operators)

Comment (biocom) [reply src/custom/components/Button/index.tsx:44]: Yes I would be in favor of that approach.

**Review by anxolin — APPROVED** [review]

File: src/custom/components/Button/index.tsx (hunk 1 above)
Comment (anxolin) [reply src/custom/components/Button/index.tsx:44]: Let's make this in another PR, since it feels independent from the scope of this one

**Pull request merged**